usaspending_api/awards/v2/lookups/lookups.py

```diff
@@ -215,6 +215,7 @@
     **other_type_mapping,
 }
 non_loan_assistance_type_mapping = {**grant_type_mapping, **direct_payment_type_mapping, **other_type_mapping}
+subcontracts_mapping = {**contract_type_mapping, **idv_type_mapping}
```

Comment:
`subcontracts_mapping` is a little misleading, perhaps `procurement_mapping` is better.